cmd/fastly-exporter/main.go

```diff
@@ -62,7 +62,7 @@ func main() {
 		fs.Var(&metricAllowlist, "metric-allowlist", "if set, only export metrics whose names match this regex (repeatable)")
 		fs.Var(&metricBlocklist, "metric-blocklist", "if set, don't export metrics whose names match this regex (repeatable)")
 		fs.DurationVar(&datacenterRefresh, "datacenter-refresh", 10*time.Minute, "how often to poll api.fastly.com for updated datacenter metadata (10m–1h)")
-		fs.DurationVar(&productRefresh, "product-refresh", 10*time.Minute, "how often to poll api.fastly.com for updated product metadata (10m–1h)")
+		fs.DurationVar(&productRefresh, "product-refresh", 60*time.Minute, "how often to poll api.fastly.com for updated product metadata (10m–24h)")
```

Comment:
What's the reasoning behind this change? If a customer enables origins or domains this will move the average response time of the exporter from 5 minutes to 30 minutes.

Reply:
Reducing load on product API was the goal. I'll revert this to 10 minutes because they are fine with the traffic the exporter sends to the product API.